pyccel/codegen/python_wrapper.py

```diff
@@ -26,22 +19,16 @@
 
 #==============================================================================
 
-PY_VERSION = sys.version_info[0:2]
-
 fortran_c_flag_equivalence = {'-Wconversion-extra' : '-Wconversion' }
```

Comment:
can we add those flags to the JSON file too?

Reply:
We can, but I'm not sure where they should go. I have tried to find the most general solution possible so translating flags only seems logical if I provide equivalences for all the flags (e.g. intel<->GNU). The solution I came up with was to split the flags into:
- Flags for compiling the source code (which must be compatible with the target language, e.g. Fortran)
- Flags for compiling the wrapper (which must be compatible with C)
This solution means that this line is dead code. I have removed it in 3cfaa14d
Do you think that is sufficient or do you think we need some kind of flag equivalence map?

Reply:
yes, it's enough.
I was before thinking about a method to simplifying compiler flags for the user so the user doesn't need to know flags differences in compilers, and we in out part creat map for the flags between compilers and suggest a friendly name for the user, but this seems hard and loosing of time.
there are a lot of important things to do than that.

Reply:
My suspicion is that either a user won't know much about flags (and therefore won't bother to specify any), or they will know the flags for the compiler they usually use and in that case they can specify that compiler.

Flags are usually useful for accelerating code (e.g. `-funroll-loops`), but these flags won't do much/anything in the wrapper anyway so, I don't think it's a huge problem if the flags are only applied to the source

Reply:
I agree with you. in my opinion the only the user has the right to apply some flags on the source files at his own risk.
for us, we may only apply some flags on the libraries we create and we tested already.

Reply:
Flags for a specific library should also be able to be done easily now by passing the flags [here](https://github.com/pyccel/pyccel/blob/ebourne_compiling/pyccel/codegen/pipeline.py#L31-L33)

Reply:
Excellent job @Emily, I always think about improving the compiling in Pyccel, but I didn't have the necessary courage ^_^

Reply:
Haha, neither did I. That's why it took a year to get done. But I think it's necessary to fix #886 cleanly. It needs to include the python files. With these improvements we just need to specify `accelerators=['python']` but without them there would have been some strange commands all over the place to find the right include files :/

Reply:
we will take care of that too.